Cliente.cs

```diff
@@ -1,10 +1,24 @@
-﻿namespace Loja_De_Roupas;
+﻿#nullable enable
+namespace Loja_De_Roupas;
 
 public class Cliente
 {
-    public int Id { get; }
-    public string Nome { get; set; }
+    private static int _id = 1;
+    public int Id { get; private init; }
+    public required string Nome { get; set; }
     public string? Email { get; set; }
     public string? Telefone { get; set; }
+    public Carrinho Carrinho { get; set; }
 
+
+    public Cliente()
+    {
+        Id = _id++;
```

Comment:
Essa abordagem , nunca sistema real não é o ideal, pois se duas chamadas chegam em paralelo, podem gerar o mesmo Id, quando é feito autoincremente assim, para ficar sequencial, normalmente eh feito isso via banco de dados, talvez nesse caso melhor abordagem é o Id fosse um GUID para gerar um exclusivo sempre que for criado o objeto